fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractIncludeRevincludeTest.java

```diff
@@ -7,6 +7,7 @@
 package com.ibm.fhir.persistence.test.common;
 
 import static com.ibm.fhir.model.type.String.string;
+import static org.junit.Assert.assertNull;
```

Comment:
let's use the testng assert 

`import static org.testng.AssertJUnit.assertNull;`

Reply:
Done.